src/core.js

```diff
@@ -303,6 +305,7 @@ function done() {
 	passed = config.stats.all - config.stats.bad;
 
 	runLoggingCallbacks( "done", {
+		skipped: Test.count - config.stats.all,
```

Comment:
I couldn't agree more. I noticed that some of these changes were really for the html reporter. I'll pill this and some of the other attributes out of this PR and address them in the next.